latest/ug/nodes/hybrid-nodes-cluster-update.adoc

```diff
@@ -2,32 +2,31 @@ include::../attributes.txt[]
 
 [.topic]
 [#hybrid-nodes-cluster-update]
-= Modify the hybrid node configuration on an existing Amazon EKS cluster
+= Enable hybrid nodes on an existing Amazon EKS cluster or modify configuration
 :info_titleabbrev: Existing cluster
 
 [abstract]
 --
-Modify hybrid nodes on an existing cluster
+Modify hybrid nodes configuration on an existing cluster
 --
 
 This topic provides an overview of the available options and describes what to consider when you add, change, or remove the hybrid nodes configuration from an Amazon EKS cluster. If you are not planning to use hybrid nodes, see <<create-cluster>>.
 
-To enable an Amazon EKS cluster to use hybrid nodes, add the IP address CIDR ranges of your on-premises network in the `RemoteNodeNetwork` configuration. EKS uses this list of CIDRs to enable connectivity between the cluster and your on-premises networks. For a full list of configuration, see the link:eks/latest/APIReference/API_UpdateClusterConfig.html[UpdateClusterConfig,type="documentation"] in the _Amazon EKS API Reference_.
+To enable an Amazon EKS cluster to use hybrid nodes, add the IP address CIDR ranges of your on-premises network in the `RemoteNodeNetwork` configuration. EKS uses this list of CIDRs to enable connectivity between the cluster and your on-premises networks. For a full list of options when updating your cluster configuration, see the link:eks/latest/APIReference/API_UpdateClusterConfig.html[UpdateClusterConfig,type="documentation"] in the _Amazon EKS API Reference_.
```

Comment:
To enable an Amazon EKS cluster to use hybrid nodes, add the IP address CIDR ranges of your on-premises node and optionally pod network in the `RemoteNetworkConfig` configuration.